docs/standard/assembly/unloadability.md

```diff
@@ -9,14 +9,14 @@ ms.date: "02/05/2019"
 
 Starting with .NET Core 3.0, the ability to load and later unload a set of assemblies is supported. In .NET Framework, custom app domains were used for this purpose, but .NET Core only supports a single default app domain.
 
-.NET Core 3.0 and later versions support unloadability through <xref:System.Runtime.Loader.AssemblyLoadContext>. You can load a set of assemblies into a collectible `AssemblyLoadContext`, execute methods in them or just inspect them using reflection, and finally unload the `AssemblyLoadContext`. That unloads the assemblies loaded into that `AssemblyLoadContext`.
+.NET Core 3.0 and later versions support unloadability through <xref:System.Runtime.Loader.AssemblyLoadContext>. You can load a set of assemblies into a collectible `AssemblyLoadContext`, execute methods in them or just inspect them using reflection, and finally unload the `AssemblyLoadContext`. That unloads the assemblies loaded into the `AssemblyLoadContext`.
```

Comment:
I think `that` was referring to the previously mentioned `AssemblyLoadContext`.

Reply:
I agree but I'm not sure what your point is?

Reply:
I meant that the original text was making sense.

Reply:
Agreed, but it had too many "thats".